Workbooks/WorkbooksMetadata.json

```diff
@@ -3682,5 +3682,26 @@
         "Platform"
       ]
     }
+  },
+  {
+    "workbookKey": "CofenseIntelligenceWorkbook",
+    "logoFileName": "CofenseTriage.svg",
+    "description": "This workbook provides visualization of Cofense Intelligence threat indicators which are ingested in the Microsoft Sentinel Threat intelligence.",
+    "dataTypesDependencies": [
+        "ThreatIntelligenceIndicator",
+        "Malware_Data"
+    ],
+    "dataConnectorsDependencies": [
+        "CofenseIntelligenceDataConnector"
+    ],
+    "previewImagesFileNames": [
+        "CofenseIntelligenceWhite1.png",
+        "CofenseIntelligenceBlack1.png"
```

Comment:
Please correct the values used in "previewImagesFileNames" parameter.
Values should be :- CofenseThreatIntelligenceBlack1.png
                               CofenseThreatIntelligenceWhite1.png
Which will redirect to the images name

Reply:
Corrected as expected. Done